imagestreams/nodejs-centos7.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
It's not clear to me why you are adding a `.json` image stream, when we already have a `.yml` image stream file for centos7 that is referenced in the community library [here](https://github.com/openshift/library/blob/master/community.yaml#L65).

Reply:
I added it since this WAS the SCL version and discussion was to move ownership to RHOAR team.

Reply:
Ahh - I guess we didn't explicitly discuss the community versions. In that case, this does make sense.

Reply:
@sspeiche To follow up on this... I suppose this means we will need to submit a pull request so that these lines are changed. 

* Remove SCL imagestream
* Update Bucharest Gold imagestream to point to this JSON file

Correct?

Reply:
Since this is the RHEL version of the image-stream, there really isn't a reference to centos-based community images.

FWIW, the community.yaml is not really used by starter or pro or ocp or origin. So either I can update in this PR the community stuff, I was focused on getting v10 to starter and pro.

Let me know if you want me to do in this PR or if should be done in a separate one.

Reply:
Sounds right